web/static/js/reducers/folders.js

```diff
@@ -0,0 +1,83 @@
+// @flow
+import * as actions from '../actions/folders'
+
+const initialState = {
+  loading: false,
+  error: null,
+  loadingFetch: false,
```

Comment:
I think we should replace `loadingFetch` with another name. It gets confused to read `loading` and `loadingFetch`?
From line 58 it looks like `loading` is actually `saving`?

Reply:
Yeah, it's confusing but it's also consistent across actions & reducers.